.xo-config.json

```diff
@@ -0,0 +1,47 @@
+{
+  "prettier": true,
+  "space": 2,
+  "plugins": ["import"],
+  "rules": {
+    "n/no-unsupported-features": "off",
+    "n/no-unsupported-features/node-builtins": "off",
+    "n/file-extension-in-import": "off",
+    "sort-imports": [
+      "error",
+      {
+        "ignoreCase": false,
+        "ignoreDeclarationSort": true,
+        "ignoreMemberSort": false,
+        "memberSyntaxSortOrder": ["none", "all", "multiple", "single"],
+        "allowSeparatedGroups": false
+      }
+    ],
+    "import/no-named-as-default": "off",
+    "import/extensions": "off",
+    "import/order": [
+      "error",
+      {
+        "groups": ["builtin", "external", "parent", "sibling", "index"],
+        "alphabetize": {
+          "order": "asc",
+          "caseInsensitive": true
+        },
+        "warnOnUnassignedImports": true,
+        "newlines-between": "never"
+      }
+    ]
+  },
+  "overrides": [
+    {
+      "files": ["sdk.mjs"],
+      "nodeVersion": ">=14"
```

Comment:
What happens if you decrease this version. Could you explain why is Node.js v14 the minimum version supported by the SDK?

Reply:
<img width="1033" alt="image" src="https://github.com/simple-icons/simple-icons/assets/8186898/b19d455a-4b85-4d13-982f-81f8a31d0b80">

Because we're using `fs/promises`.

Reply:
Great. So can we add a check of the major Node.js version used to the start of the SDK file? By that approach can have something like `node.engine` checks for each export.

Reply:
How about publishing the SDK as a new package?

Reply:
Yep, that can be another option. Something like `@simple-icons/sdk`. But that would be a breaking change that we can postpone for the v12.

I'm fine if you don't want add the check here but still thinking that the `node.engine` is worth for the main package.

Reply:
Sure. I will add the `node.engine` back. And it should be `>=12.16.0` due to https://nodejs.org/api/packages.html#conditional-exports.